src/AciContractCallEncoder.js

```diff
@@ -72,20 +72,49 @@ class AciContractCallEncoder {
      * const data = encoder.decodeCall('Test', 'test_string', 'cb_KxHwzCuVGyl3aG9vbHltb2x5zwMSnw==')
      * console.log(`Decoded data: ${data}`)
      * // Outputs:
-     * // Decoded data: ["whoolymoly"]
+     * // Decoded data: { functionId: "f0cc2b95", args: ["whoolymoly"] }
      *
      * @param {string} contract - The contract name as defined in the ACI.
      * @param {string} funName - The function name as defined in the ACI.
      * @param {string} data - Encoded calldata in canonical format.
      * @returns {string} Decoded data
     */
     decodeCall(contract, funName, data) {
-        const {types, _required} = this._typeResolver.getCallTypes(contract, funName)
+        const {types} = this._typeResolver.getCallTypes(contract, funName)
         const calldataType = FateTypeCalldata(funName, types)
 
         return this._byteArrayEncoder.decodeWithType(data, calldataType)
     }
 
+    /**
+     * Decodes contract calldata without function name
+     *
+     * @example
+     * const data = encoder.decodeCall('cb_KxHwzCuVGyl3aG9vbHltb2x5zwMSnw==')
+     * console.log(`Decoded data: ${data}`)
+     * // Outputs:
+     * // Decoded data: {
+     * //   contractName: 'Test',
+     * //   functionName: 'test_string',
+     * //   functionId: "f0cc2b95",
+     * //   args: ["whoolymoly"],
+     * // }
+     *
+     * @param {string} data - Encoded calldata in canonical format.
+     * @returns {object} Decoded data
+    */
+    decodeCallWithoutFunctionName(data) {
```

Comment:
I prefer to have a single responsibility methods, thus something like `decodeFunction` without returning and decoding the actual `data` as it already exists as functionality